PluralKit.Bot/Commands/MemberProxy.cs

```diff
@@ -17,6 +17,8 @@ ProxyTag ParseProxyTags(string exampleProxy)
             if (prefixAndSuffix.Length == 1) prefixAndSuffix = prefixAndSuffix[0].Split("TEXT");
             if (prefixAndSuffix.Length < 2) throw Errors.ProxyMustHaveText;
             if (prefixAndSuffix.Length > 2) throw Errors.ProxyMultipleText;
+            if (prefixAndSuffix[0].length > 64) throw Errors.ProxyTooBig;
+            if (prefixAndSuffix[1].length > 64) throw Errors.ProxyTooBig;
```

Comment:
Please change this to use the same check (and return the same error) as the proxy tag length check [later in this file (in the "add" subcommand)](https://github.com/PluralKit/PluralKit/blob/528da39e66447e0fc8eab2e993516c12a6585bf6/PluralKit.Bot/Commands/MemberProxy.cs#L70-L72)